kobo/apps/openrosa/apps/logger/models/attachment.py

```diff
@@ -78,6 +77,9 @@ class Attachment(AbstractTimeStampedModel, models.Model, AudioTranscodingMixin):
         db_index=True,
     )
 
+    date_created = models.DateTimeField(null=True, blank=True)
+    date_modified = models.DateTimeField(null=True, blank=True)
```

Comment:
We need to populate those two fields for new attachments. 
I was going to say, let's save them in the `.save()` method (temporarily), but finally, we should keep what you did at the beginning.

```python
class Attachment(AbstractTimeStampedModel, AudioTranscodingMixin):
     ...
     # override those two fields from  AbstractTimeStampedModel to ensure they are nullable and 
     # not backfilled with current timestamp when the migration runs.
     # ToDo remove in future release, when TASK-1534 has completed
    date_created = models.DateTimeField(null=True, blank=True)
    date_modified = models.DateTimeField(null=True, blank=True)
```